src/quotes.ts

```diff
@@ -248,9 +248,14 @@ export const quotes: Quote[] = [
 	},
 	{
 		quote: `Put your hand on a hot stove for a minute, and it seems like an hour. Sit with a pretty girl for an hour and it seems like a minute. That is relativity.`,
-		author: `Albert Einstein`,
+		author: `Paulo Coelho`,
```

Comment:
By `author`, I mean the one who actually said that and not the one who quoted it @Praveen007-K